storage_service/locations/models/async.py

```diff
@@ -33,7 +34,14 @@ class Async(models.Model):
 
     @property
     def result(self):
-        return cPickle.loads(self._result)
+        result = self._result
+        # Convert memoryview (PY3) to str.
+        if sys.version_info[0] == 3 and isinstance(result, memoryview):
+            result = str(result)
+        # Convert buffer (PY2) to str.
+        if sys.version_info[0] == 2 and isinstance(result, buffer):
+            result = str(result)
```

Comment:
It looks like you could simplify using this by using `from django.utils import six`:

    if isinstance(result, six.memoryview):
        result = str(result)

Reply:
Cf. https://docs.djangoproject.com/en/2.0/_modules/django/utils/six/